### PR TITLE
wrap the main fiber methods in the server struct and allow plugins to configure fiber.Config

### LINKED ISSUE #24: wrap the main fiber methods in the server struct
to add a GET endpoint, we need to call the App () method.

it would be interesting to be able to do it this way:

```
srv := fiber.NewServer(ctx)

srv.Get(c.App.Endpoint.Helloworld, handler.Get)
srv.Serve(ctx)
```

only plugins that interact with the fiber instance are also allowed. It would be interesting to modify the config.

### PR DESCRIPTION
**Pull request by jpfaria**
fixes #24

### CODE REVIEW

File: gofiber/fiber.v2/config.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ import (
 const (
 	root                      = "ignite.fiber"
 	port                      = root + ".port"
+	tp                        = root + ".type"
```
Comment (aleal) [inline gofiber/fiber.v2/config.go:11]: maybe  **appType** instead of **tp**. More readable and meaningful i think

Comment (jpfaria) [reply gofiber/fiber.v2/config.go:11]: it is a private constant. I think it’s okay to be small.

**Review by aleal — APPROVED** [review]
I think eveything is fine. One thing, in general, I mean not just about this PR, all libs should avoid the use of Get on getters, except to solve a naming conflict. This is more "go like" programming. Example: __getAllowedHeaders__ could be __allowedHeaders__ 
Maybe we could open an issue to address these kind of situations.  Or may be I am just being narcissistic...

File: gofiber/fiber.v2/config.go (hunk 1 above)
Comment (aleal) [reply gofiber/fiber.v2/config.go:11]: ok

**Pull request merged**